diff --git a/slock.c b/slock.c
index b2f14e3..be7e5f9 100644
--- a/slock.c
+++ b/slock.c
@@ -156,6 +156,7 @@ readpw(Display *dpy, struct xrandr *rr, struct lock **locks, int nscreens,
 			    IsPFKey(ksym) ||
 			    IsPrivateKeypadKey(ksym))
 				continue;
+#if 0
 			switch (ksym) {
 			case XK_Return:
 				passwd[len] = '\0';
@@ -197,6 +198,10 @@ readpw(Display *dpy, struct xrandr *rr, struct lock **locks, int nscreens,
 				}
 				oldc = color;
 			}
+#else
+			if (ksym == XK_Return && (ev.xkey.state & Mod4Mask))
+				running = 0;
+#endif
 		} else if (rr->active && ev.type == rr->evbase + RRScreenChangeNotify) {
 			rre = (XRRScreenChangeNotifyEvent*)&ev;
 			for (screen = 0; screen < nscreens; screen++) {
